modules/ROOT/pages/supported-cluster-config.adoc

```diff
@@ -124,7 +124,65 @@ The following apply to both 4-node and 7-node configurations.
 
 === Anypoint Monitoring
 
-For limitations onf using Anypoint Monitoring with Anypoint Platform PCE see xref:anypoint-monitoring.adoc[Anypoint Monitoring on Anypoint Platform PCE]
+Anypoint Monitoring is an optional, add-on component. See xref:anypoint-monitoring.adoc[Anypoint Monitoring on Anypoint Platform PCE]. The following limitations apply when using Anypoint Monitoring with Anpoint Platform PCE.
+
+==== Enhanced API Analytics Monitoring Limits
+
+* API Summary Metrics
+** Retention: 1 year
+
+* API IP Metrics
+** Number of IP addresses per API per minute: restricted to 20 unique IP addresses.
```

Comment:
Should we use the phrase "restricted to" in the limits? It might raise questions about whether those restrictions could be lifted.

Reply:
Changed to "limited."